cmd/nf-dump/main.go

```diff
@@ -40,16 +40,21 @@ func main() {
 		log.Fatal(err)
 	}
 
+	if err = server.SetReadBuffer(262144); err != nil {
```

Comment:
Mine is Gentoo, I think it will "round it" to max value allowed (as in rmem_max), but need to check that. 
I have buffers tuned on my system, but you are right, on default system it might be not beneficial, because defaults values looks like:
net.core.rmem_default = 212992
net.core.rmem_max = 212992

but i didn't added code to check proc value, to not make it Linux specific, and seems there is no cross-platform API to get current max value.